mne/io/ctf/tests/test_ctf.py

```diff
@@ -197,4 +197,15 @@ def test_read_ctf():
     assert_raises(ValueError, read_raw_ctf,
                   op.join(ctf_dir, ctf_fname_continuous), 'foo')
 
+
+@testing.requires_spm_data
```

Comment:
this should probably be `spm_face.requires_spm_data`. `testing` is the namespace for the `testing` dataset, not all things related to testing